tests/specs/node/node_test_module/test.js

```diff
@@ -79,6 +79,36 @@ test("reject fail", () => {
   return Promise.reject(new Error("rejected from reject fail"));
 });
 
+suite("suite", () => {
+  test("test 1", () => {});
+  test("test 2", () => {});
+
+  suite("sub suite 1", () => {
+    test("nested test 1", () => {});
+    test("nested test 2", () => {});
+  });
+
+  suite("sub suite 2", () => {
+    test("nested test 1", () => {});
+    test("nested test 2", () => {});
+  });
+});
+
+describe("describe", () => {
+  it("test 1", () => {});
+  it("test 2", () => {});
+
+  describe("sub describe 1", () => {
+    it("nested test 1", () => {});
+    it("nested test 2", () => {});
+  });
+
+  describe("sub describe 2", () => {
+    it("nested test 1", () => {});
```

Comment:
Would be nice to have failing test cases here to ensure errors are properly caught and reported